metpy/calc/thermo.py

```diff
@@ -840,3 +842,180 @@ def relative_humidity_from_specific_humidity(specific_humidity, temperature, pre
     return (100 * units.percent *
             mixing_ratio_from_specific_humidity(specific_humidity) /
             saturation_mixing_ratio(pressure, temperature))
+
+
+@exporter.export
+@check_units('[temperature]', '[pressure]', '[temperature]')
+def get_isentropic_pressure(isentlevs, lev, tmp, *args, **kwargs):
+    r"""Perform isentropic analysis with data in isobaric coordinates.
+
```

Comment:
Would be good to have additional verbiage here describing briefly describe what isentropic analysis is (i.e. "interpolate to levels of constant potential temperature")